docs/release-notes/0.10.4.md

```diff
@@ -6,6 +6,7 @@
 * `AnnData.__sizeof__()` support for backed datasets {pr}`1230` {user}`Neah-Ko`
 * `adata[:, []]` now returns an `AnnData` object empty on the appropriate dimensions instead of erroring {pr}`1243` {user}`ilan-gold`
 * `adata.X[mask]` works in newer `numpy` versions when `X` is `backed` {pr}`1255` {user}`ilan-gold`
+* `BaseCompressedSparseDataset`'s `indptr` is cached {pr}`1266` {user}`ilan-gold`
```

Comment:
Quick scan, didn't see any "experimental" headers before so just put it under performance.  My first reaction was that we were releasing an experimental feature when I read you comment about the header, which is not the impression we want to give for a patch release.

Reply:
There's nothing special about the headers, it's just text.

I was thinking that it could go under experimental since it's a bit of an API change, which otherwise wouldn't be allowed in a bug fix release. But I don't feel super strongly about this here.